doc/howtos/software-update-server.rst

```diff
@@ -0,0 +1,57 @@
+.. _software-update-server:
+
+Software Update: Building an Ostro |trade| OS  Repository
+#########################################################
+
+This technical note provides instructions supporting the creation
+of a repository for feeding updates to devices based on Ostro OS.
+
```

Comment:
My feeling based on discussions with people who don't know swupd is that they may think 'swupd-server' is the web server providing the updates. It is mentioned in this doc that swupd-server is a tool used to create the content but I'm thinking it would make sense to explicitly mention that if the updates are published online a web server is also needed.